classes/OccurrenceLabel.php

```diff
@@ -626,77 +663,104 @@ public function deleteLabelFormat($group, $labelIndex){
 		return $status;
 	}
 
-	private function saveGlobalJson($formatArr){
+	private function saveGlobalJson($formatArr) {
 		$status = false;
-		$jsonStr = "<?php\n ".'$LABEL_FORMAT_JSON = \''.json_encode($formatArr,JSON_PRETTY_PRINT | JSON_HEX_APOS)."'; \n?>";
-		if($fh = fopen($GLOBALS['SERVER_ROOT'].'/content/collections/reports/labeljson.php','w')){
-			if(!fwrite($fh,$jsonStr)){
-				$this->errorArr[] = 'ERROR saving label format to global file ';
-				$status = false;
-			}
-			fclose($fh);
-		}
-		else{
-			$this->errorArr[] = 'ERROR saving label format: unable opening/creating labeljson.php for writing';
-			$status = false;
+
+		$jsonDynProps = json_encode($formatArr, JSON_PRETTY_PRINT | JSON_HEX_APOS);
+		$attributeName = 'LabelFormatJson';
+		$checkSql = "SELECT COUNT(*) FROM adminconfig WHERE attributeName = ?";
+		if ($checkStmt = $this->conn->prepare($checkSql)) {
+			$checkStmt->bind_param('s', $attributeName);
+			$checkStmt->execute();
+			$checkStmt->bind_result($count);
+			$checkStmt->fetch();
+			$checkStmt->close();
+
+			if ($count > 0) {
```

Comment:
Handles the case where the row already exists in the adminconfig and just needs to be updated.